src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -362,7 +362,7 @@ class VAMap extends Component {
         bounds._northEast.lat,
       ];
     }
-
+    if (!currentQuery.facilityType) return;
```

Comment:
Can we move this to the top of the function to avoid unnecessary computation?

Reply:
done